### PR TITLE
DOMA-1522 section unit types

### PR DESCRIPTION
**Pull request by sitozzz**
Adding unitType field to section unit.

- [x] Add `unitType` field to `Ticket` model as keystone `Select` type
- [x] Group by `BuildingUnitType` at `<UnitInfo />`
- [x] Display group related prefix `<TicketTable />` and ticket retrieve view

image

image

image

image

### CODE REVIEW

File: apps/condo/domains/property/components/panels/Builder/MapConstructor.ts [hunk 1]
```diff
@@ -903,13 +912,15 @@ class MapEdit extends MapView {
         if (unitIndex.unit === -1) {
             return
         }
+        console.log(unit)
```
Comment (AntonAL) [inline apps/condo/domains/property/components/panels/Builder/MapConstructor.ts:915]:
```suggestion

```

**Review by AntonAL — APPROVED** [review]

**Review by pahaz — APPROVED** [review]

**Review by dkoviazin — APPROVED** [review]

File: apps/condo/domains/property/components/panels/Builder/MapJsonSchema.json [hunk 2]
```diff
@@ -62,7 +62,10 @@
                                             "unitType": {
                                                 "enum": [
                                                     "flat",
-                                                    "parking"
+                                                    "parking",
```
Comment (pahaz) [inline apps/condo/domains/property/components/panels/Builder/MapJsonSchema.json:65]: These values should be imported from consts

File: apps/condo/domains/property/gql.js [hunk 3]
```diff
@@ -26,6 +26,9 @@ const PROPERTY_MAP_GRAPHQL_TYPES = `
     enum BuildingUnitType {
         parking
         flat
+        apartment
```
Comment (pahaz) [inline apps/condo/domains/property/gql.js:29]: These values should be imported from consts

Comment (sitozzz) [reply apps/condo/domains/property/gql.js]: Fixed this

**Pull request merged**